app/scripts/services/builds.js

```diff
@@ -70,10 +71,16 @@ angular.module("openshiftConsole")
         namespace: buildConfig.metadata.namespace
       };
       return DataService.create("buildconfigs/instantiate", buildConfig.metadata.name, req, context).then(function(build) {
+        var details;
         var displayName = getBuildDisplayName(build, buildConfig.metadata.name);
+        var runPolicy = _.get(buildConfig, 'spec.runPolicy');
+        if (runPolicy === 'Serial' || runPolicy === 'SerialLatestOnly') {
+          details = "The build has been queued with run policy " + camelToLower(runPolicy) + ".";
```

Comment:
sentence seems like it reads strangely to me

maybe "This build has been queued under a serial run policy."  and not worry about the distinction between serial and serial latest only?  The important thing is that its not parallel.

alternatively "This build is configured to run serially and has been added to the queue."